include/boost/geometry/io/wkt/detail/prefix.hpp

```diff
@@ -42,6 +47,11 @@ struct prefix_linestring
     static inline const char* apply() { return "LINESTRING"; }
 };
 
+struct prefix_polyhedron
+{
+	static inline const char* apply() { return "POLYHEDRON"; }
```

Comment:
as far as I can see it will look like
```
POLYHEDRALSURFACE Z ( PATCHES
    ((0 0 0, 0 1 0, 1 1 0, 1 0 0, 0 0 0)),
    ((0 0 0, 0 1 0, 0 1 1, 0 0 1, 0 0 0)),
    ((0 0 0, 1 0 0, 1 0 1, 0 0 1, 0 0 0)),
    ((1 1 1, 1 0 1, 0 0 1, 0 1 1, 1 1 1)),
    ((1 1 1, 1 0 1, 1 0 0, 1 1 0, 1 1 1)),
    ((1 1 1, 1 1 0, 0 1 0, 0 1 1, 1 1 1))
  )
```
(see wikipedia), I don't see anything of this reflected in your code.

Before accepting a pull request with completely new elements like this, it should be a complete part, we should be able to understand the intentions, and there should be a working sample, reading a polyhedral surface from such a WKT and writing it back.

And there is more: it should be concept based.

The `REGISTER` functionality can be omitted from the first PR - it is too far ahead to judge how it should look like.

Reply:
> (see wikipedia), I don't see anything of this reflected in your code.

yeah, I am working on it. Once I fix the PolyhedralSuface header file, I will complete the wkt read and write.

Reply:
Do, I need to include the term `PATCHES` in wkt read or write, as it is self-understood from the term `POLYHEDRALSURFACE`.

Reply:
Interesting, I'm not sure where does the `PATCHES` come from in the Wikipedia, but it does not look right.
AFAIS, there is no such WKT tag specified/presented in the OGC SFS 1.2.1

AFAIK, neither PostGIS nor GDAL/OGR implementation reads/writes it. 

From personal experience, I've never used it in any WKT implementations I've done myself.

Reply:
Indeed, I see. It's probably not from the specs indeed. So please skip the `PATCHES` keyword.

So it will then be something like 
```
POLYHEDRALSURFACE(
	((0 0 0, 0 0 1, 0 1 1, 0 1 0, 0 0 0)),
	((0 0 0, 0 1 0, 1 1 0, 1 0 0, 0 0 0)),
	((0 0 0, 1 0 0, 1 0 1, 0 0 1, 0 0 0)),
	((1 1 0, 1 1 1, 1 0 1, 1 0 0, 1 1 0)),
	((0 1 0, 0 1 1, 1 1 1, 1 1 0, 0 1 0)),
	((0 0 1, 1 0 1, 1 1 1, 0 1 1, 0 0 1)))
```
(taken from https://postgis.net/docs/ST_GeomFromEWKT.html)